extractor/src/main/java/org/schabi/newpipe/extractor/services/media_ccc/extractors/MediaCCCRecentKioskExtractor.java

```diff
@@ -0,0 +1,79 @@
+package org.schabi.newpipe.extractor.services.media_ccc.extractors;
+
+import com.grack.nanojson.JsonObject;
+import org.schabi.newpipe.extractor.exceptions.ParsingException;
+import org.schabi.newpipe.extractor.localization.DateWrapper;
+import org.schabi.newpipe.extractor.stream.StreamInfoItemExtractor;
+import org.schabi.newpipe.extractor.stream.StreamType;
+
+import javax.annotation.Nullable;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class MediaCCCRecentKioskExtractor implements StreamInfoItemExtractor {
+
+    private final JsonObject event;
+
+    public MediaCCCRecentKioskExtractor(final JsonObject event) {
+        this.event = event;
+    }
+
+    @Override
+    public String getName() throws ParsingException {
+        return event.getString("title");
+    }
+
+    @Override
+    public String getUrl() throws ParsingException {
+        return event.getString("frontend_link");
+    }
+
+    @Override
+    public String getThumbnailUrl() throws ParsingException {
+        return event.getString("thumb_url");
+    }
+
+    @Override
+    public StreamType getStreamType() throws ParsingException {
+        return StreamType.VIDEO_STREAM;
+    }
+
+    @Override
+    public boolean isAd() throws ParsingException {
+        return false;
+    }
+
+    @Override
+    public long getDuration() throws ParsingException {
+        return event.getInt("duration");
+    }
+
+    @Override
+    public long getViewCount() throws ParsingException {
+        return event.getInt("view_count");
+    }
+
+    @Override
+    public String getUploaderName() throws ParsingException {
+        return "";
+    }
+
+    @Override
+    public String getUploaderUrl() throws ParsingException {
+        return event.getString("conference_url");
```

Comment:
this returns an api link. Are you sure it is ok?
Without an api request you can build the uploader url, get the last part from this link and add it here: `https://media.ccc.de/c/<uploader_id>`

Reply:
done the problem is still that the uploader is not shown, because we don't have the uploader name / conference name.
We could make an API request to get the conference name and store the value in some static map to reduce the number of requests.

Reply:
I now get the conference URL.
Disadvantage: ~560kB are downloaded, 7 requests made. The number of requests and the size of the retrieve JSON documents can vary extremely. This is because every conference documents contains massive data about its events.
@Stypox @B0pol @TheAssassin